handlers/handler-loadbalance/src/test/java/org/apache/servicecomb/loadbalance/filter/TestIsolationServerListFilter.java

```diff
@@ -60,29 +63,26 @@ public static void classTeardown() {
     Deencapsulation.setField(ConfigurationManager.class, "instance", null);
     Deencapsulation.setField(ConfigurationManager.class, "customConfigurationInstalled", false);
     Deencapsulation.setField(DynamicPropertyFactory.class, "config", null);
+    ArchaiusUtils.resetConfig();
   }
 
   @Before
   public void setUp() throws Exception {
     IsolationServerListFilter = new IsolationServerListFilter();
     loadBalancerStats = new LoadBalancerStats("loadBalancer");
-
-    AbstractConfiguration configuration =
-        (AbstractConfiguration) DynamicPropertyFactory.getBackingConfigurationSource();
-    configuration.clearProperty("cse.loadbalance.isolation.enabled");
-    configuration.addProperty("cse.loadbalance.isolation.enabled",
+    ArchaiusUtils.setProperty("cse.loadbalance.isolation.enabled",
         "true");
-    configuration.clearProperty("cse.loadbalance.isolation.enableRequestThreshold");
-    configuration.addProperty("cse.loadbalance.isolation.enableRequestThreshold",
+    ArchaiusUtils.setProperty("cse.loadbalance.isolation.enableRequestThreshold",
         "3");
 
     taskList = new ArrayList<>();
-    EventManager.register(new Object() {
+    receiveEvent = new Object() {
       @Subscribe
       public void onEvent(AlarmEvent isolationServerEvent) {
         taskList.add(isolationServerEvent);
       }
-    });
+    };
+    EventManager.register(receiveEvent);
```

Comment:
EventManager is global instance
where is the clear action?

and it's better not do this
also had wrote this days ago

Reply:
done,save a reference in filter when UT create a new instance for it.